Tasks/Maksim/Task5/query05.sql

```diff
@@ -0,0 +1,25 @@
+--Query  5
```

Comment:
not working anymore Emp should be excluded
final result is not corrent cuz there are plenty of Emp that has orders in 2012 in the output